references/recognition/train_pytorch.py

```diff
@@ -127,6 +127,7 @@ def fit_one_epoch(model, train_loader, batch_transforms, optimizer, scheduler, m
             scaler.update()
         else:
             train_loss = model(images, targets)["loss"]
+
```

Comment:
remove this empty line please

Reply:
@nikokks

Reply:
and run `make style` please

Reply:
after you have installed docts dev with `pip3 install -e .[dev]`

Reply:
@nikokks still to remove this line please xD